ckanext/pages/actions.py

```diff
@@ -110,7 +112,7 @@ def _pages_list(context, data_dict):
         pg_row = {'title': pg.title,
                   'content': pg.content,
                   'name': pg.name,
-                  'publish_date': pg.publish_date,
+                  'publish_date': pg.publish_date.isoformat() if pg.publish_date else None,
```

Comment:
@perceptron-XYZ this is needed for compatibility with the API, does this change affect the controller's use of this code?